### DISCUSSION TITLE
Hungary ELZETT Key Support

### DISCUSSION THREAD

**Discussion by Tuyeee** [zinongli/KeyCopier#18] [Ideas]
Hello
I would like to know what would be needed to add other types of keys to the program?
I am thinking of the ELZETT type lock key standardized in Hungary. 5 pins, max 7 levels per pin.

**Comment by HonestLocksmith** [comment]
ELZ1?

**Reply by Tuyeee** [reply]
HI

I think the is smaller than the base model (it will be a smaller padlock key)

I'll give you a link where you can see the names of some manufacturers

https://autokulcs.hu/lakaskulcsok-egyeb-/5767-gyari-bp-elzett-el1c-10-errebi-ez10-jma-elz-8d-orion-keyline-epueletkulcs-lakaskulcs-100005990001.html

Other company model names:
• SILCA: ELIC • ERREBI: EZ1O • JMA: ELZ-8D • ELZETT: BP ELZETT • HD: XH1074

![20250121_195141](https://github.com/user-attachments/assets/2c7ae183-ddc5-433d-8be5-bbc4c334a63c)

![uj_seclox_elzett-EL26_3](https://github.com/user-attachments/assets/9d766f28-a8e5-4891-ad9c-257e55ef4822)
![uj_seclox_elzett-EL26_2](https://github.com/user-attachments/assets/94aa1d11-b432-45bd-ac78-1ae34fb5db05)
![uj_seclox_elzett-EL26_1](https://github.com/user-attachments/assets/4ea47e3f-5650-46cf-9e54-c8f83e7391ea)

**Comment by HonestLocksmith** [comment]
![20250122_000612](https://github.com/user-attachments/assets/f9c1b5e3-0d2d-4193-9926-9e41f65a5f70)

It's added for the next PR.
